nixos/modules/services/misc/mesos-slave.nix

```diff
@@ -43,6 +51,12 @@ in {
         type = types.bool;
       };
 
+      withDocker = mkOption {
```

Comment:
@rushmorem I'm still interested too, just haven't had time. Will pick it up, sorry for the delay.